applicationconfig/client/src/main/java/com/azure/applicationconfig/models/ConfigurationSetting.java

```diff
@@ -117,50 +117,20 @@ public String etag() {
         return etag;
     }
 
-    /**
-     * Sets the etag.
-     * @param etag etag
-     * @return ConfigurationSetting object itself
-     */
-    public ConfigurationSetting withEtag(String etag) {
-        this.etag = etag;
-        return this;
-    }
```

Comment:
Earlier on I noted you had public API that would accept a ConfigurationSetting and also noted that the etag was going to be checked in those calls. How is the user supposed to specify the etag for those cases?

Reply:
During our service team talks, this field is populated by service calls by default.. So they expect users to "add", "get" or update some setting and use that object to make calls for subsequent operations. 🤔